## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 427578`

Logs: https://github.com/GaetanLepage/nixpkgs-review-gha/actions/runs/16456553971


---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>omnisharp-roslyn</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>omnisharp-roslyn</summary>
<pre>/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102: Unable to find package Microsoft.Extensions.FileProviders.Physical with version (>= 9.0.0) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102:   - Found 1 version(s) in _nix [ Nearest version: 8.0.0 ] [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102: Unable to find package Microsoft.Extensions.FileSystemGlobbing with version (>= 9.0.0) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102:   - Found 1 version(s) in _nix [ Nearest version: 8.0.0 ] [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error : The given key 'Microsoft.Extensions.Configuration' was not present in the dictionary. [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at System.Collections.Generic.Dictionary`2.get_Item(TKey key) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.LockFileBuilder.GetLibraryDependenciesForCentralTransitiveDependencies(RestoreTargetGraph targetGraph, TargetFrameworkInformation targetFrameworkInformation)+MoveNext() [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.LockFileBuilder.AddCentralTransitiveDependencyGroupsForPackageReference(PackageSpec project, LockFile lockFile, IEnumerable`1 targetGraphs) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.LockFileBuilder.CreateLockFile(LockFile previousLockFile, PackageSpec project, IEnumerable`1 targetGraphs, IReadOnlyList`1 localRepositories, RemoteWalkContext context, LockFileBuilderCache lockFileBuilderCache) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreCommand.BuildAssetsFile(LockFile existingLockFile, PackageSpec project, IEnumerable`1 graphs, IReadOnlyList`1 localRepositories, RemoteWalkContext contextForProject) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreCommand.ExecuteAsync(CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.ExecuteAsync(RestoreSummaryRequest summaryRequest, CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.ExecuteAndCommitAsync(RestoreSummaryRequest summaryRequest, IRestoreProgressReporter progressReporter, CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.CompleteTaskAsync(List`1 restoreTasks) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.RunAsync(IEnumerable`1 restoreRequests, RestoreArgs restoreArgs, CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.RunAsync(RestoreArgs restoreContext, CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Build.Tasks.BuildTasksUtility.RestoreAsync(DependencyGraphSpec dependencyGraphSpec, Boolean interactive, Boolean recursive, Boolean noCache, Boolean ignoreFailedSources, Boolean disableParallel, Boolean force, Boolean forceEvaluate, Boolean hideWarningsAndErrors, Boolean restorePC, Boolean cleanupAssetsForUnsupportedProjects, ILogger log, CancellationToken cancellationToken) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/xd9xy9mf4wmmlcdk51kzvsrnazqsajfz-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Build.Tasks.Console.MSBuildStaticGraphRestore.RestoreAsync(String entryProjectFilePath, IDictionary`2 globalProperties, IReadOnlyDictionary`2 options) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]</pre>
</details>
</details>


---
### `aarch64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>omnisharp-roslyn</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>omnisharp-roslyn</summary>
<pre>/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102:   - Found 1 version(s) in _nix [ Nearest version: 8.0.0 ] [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102: Unable to find package Microsoft.Extensions.FileProviders.Physical with version (>= 9.0.0) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102:   - Found 1 version(s) in _nix [ Nearest version: 8.0.0 ] [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102: Unable to find package Microsoft.Extensions.FileSystemGlobbing with version (>= 9.0.0) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/build/source/src/OmniSharp.Script/OmniSharp.Script.csproj : error NU1102:   - Found 1 version(s) in _nix [ Nearest version: 8.0.0 ] [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error : The given key 'NuGet.ProjectModel' was not present in the dictionary. [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at System.Collections.Generic.Dictionary`2.get_Item(TKey key) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.LockFileBuilder.GetLibraryDependenciesForCentralTransitiveDependencies(RestoreTargetGraph targetGraph, TargetFrameworkInformation targetFrameworkInformation)+MoveNext() [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.LockFileBuilder.AddCentralTransitiveDependencyGroupsForPackageReference(PackageSpec project, LockFile lockFile, IEnumerable`1 targetGraphs) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.LockFileBuilder.CreateLockFile(LockFile previousLockFile, PackageSpec project, IEnumerable`1 targetGraphs, IReadOnlyList`1 localRepositories, RemoteWalkContext context, LockFileBuilderCache lockFileBuilderCache) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreCommand.ExecuteAsync(CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.ExecuteAsync(RestoreSummaryRequest summaryRequest, CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.ExecuteAndCommitAsync(RestoreSummaryRequest summaryRequest, IRestoreProgressReporter progressReporter, CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.CompleteTaskAsync(List`1 restoreTasks) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.RunAsync(IEnumerable`1 restoreRequests, RestoreArgs restoreArgs, CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Commands.RestoreRunner.RunAsync(RestoreArgs restoreContext, CancellationToken token) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Build.Tasks.BuildTasksUtility.RestoreAsync(DependencyGraphSpec dependencyGraphSpec, Boolean interactive, Boolean recursive, Boolean noCache, Boolean ignoreFailedSources, Boolean disableParallel, Boolean force, Boolean forceEvaluate, Boolean hideWarningsAndErrors, Boolean restorePC, Boolean cleanupAssetsForUnsupportedProjects, ILogger log, CancellationToken cancellationToken) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]
/nix/store/h9if4bg6jy874piwzkclzxfrw1wip20b-dotnet-sdk-8.0.412/share/dotnet/sdk/8.0.412/NuGet.RestoreEx.targets(19,5): error :    at NuGet.Build.Tasks.Console.MSBuildStaticGraphRestore.RestoreAsync(String entryProjectFilePath, IDictionary`2 globalProperties, IReadOnlyDictionary`2 options) [/build/source/src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj]</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>omnisharp-roslyn</li>
  </ul>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>omnisharp-roslyn</li>
  </ul>
</details>

--- a/pkgs/by-name/om/omnisharp-roslyn/package.nix
+++ b/pkgs/by-name/om/omnisharp-roslyn/package.nix
@@ -13,13 +13,13 @@ in
 let
   finalPackage = buildDotnetModule rec {
     pname = "omnisharp-roslyn";
-    version = "1.39.13";
+    version = "1.39.14";
 
     src = fetchFromGitHub {
       owner = "OmniSharp";
       repo = "omnisharp-roslyn";
       tag = "v${version}";
-      hash = "sha256-/U7zpx0jAnvZl7tshGV7wORD/wQUKYgX1kADpyCXHM4=";
+      hash = "sha256-b/3bp/HyInGg6sjb0/q552jSq2EZWDhFs5xApV31BL4=";
     };
 
     projectFile = "src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj";

